_layouts/default.html

```diff
@@ -80,6 +80,9 @@
           <a href="{{ site.baseurl }}/wifi.html">wifi</a>
         </li>
         <li>
+          <a href="{{ site.baseurl }}/ap.html">access point</a>
```

Comment:
Since it is related to networking, I thought it would make sense to follow wifi. From the current logic, it seems like full project tutorials follow the modules section. But since it is kind of part 1 for the webserver, it could also make sense before that.